lib/siftsciex/verification/response.ex

```diff
@@ -24,7 +24,7 @@ defmodule Siftsciex.Verification.Response do
         status: 50
       }
   """
-  @spec process(String.t, String.t) :: __MODULE__.t
+  @spec process(String.t(), String.t()) :: CheckResponse.t()
```

Comment:
Should this be `CheckResponse.t() | SendResponse.t()`?

Reply:
Yes. I'll close this in favor of https://github.com/TheRealReal/siftsciex/pull/2